authors/apps/authentication/tests/test_users.py

```diff
@@ -0,0 +1,54 @@
+from rest_framework import status
```

Comment:
You can divide the tests and create an `api` folder inside `tests` folder and place this file there. The same will happen for the `user model`